bianca_pipes.rb

```diff
@@ -0,0 +1,202 @@
+##### Wave_1 #####
+
+#create SolarSystem class
+
+class SolarSystem
+
+  #use @planets instance variable
+  def initialize(planets)
+    @planets = Array.new(planets)
+  end
+
+  #method to add planet to list
+
+  def add(planet)
+    @planet = planet
+    @planets << planet
+    return @planet
+  end
+
+  #method to return, not print list of planets as String
+  def list
+    num = 0
+    planet_list = ""
+    @planets.each do |planet|
+      num += 1
+      planet_list += "#{num}. #{planet}\n"
+    end
+    ## Not sure why return does not work here instead of puts ##
+    puts planet_list
+  end
+end
+
+milky_way = SolarSystem.new(milky_way)
+
+milky_way.add("Mercury")
+milky_way.add("Venus")
+milky_way.add("Earth")
+
+milky_way.list
+
+milky_way.add("Hoth")
+
+milky_way.list
+
+class SolarSystem
+
+def initialize(planets, planet_name, mass, orbit, tilt, rotation, num_of_moons)
+  @planets = planets
+  @planets = Hash.new
+  @planet_name, @mass, @orbit, @tilt, @rotation, @num_of_moons  = planet_name, mass, orbit, tilt, rotation, num_of_moons
+  @planets[:name] = planet_name
+  @planets[:mass] = mass
+  @planets[:orbit] = orbit
+  @planets[:tilt] = tilt
+  @planets[:rotation] = rotation
+  @planets[:num_of_moons] = num_of_moons
+end
+
+def add_hash(planet_name, mass, orbit, tilt, rotation, num_of_moons)
+  @planets[:name] = planet_name
+  @planets[:mass] = mass
+  @planets[:orbit] = orbit
+  @planets[:tilt] = tilt
+  @planets[:rotation] = rotation
+  @planets[:num_of_moons] = num_of_moons
+end
+
+#return list of planets as string
+def return_planets
+  planet_hash = ""
+  @planets.each do |key, value|
+    planet_hash += "#{key}: #{value}\n"
+  end
+  puts planet_hash
+end
+end
+
+milky_way = SolarSystem.new("milkyway", "Earth", 34, 5, 6, 7, 1)
+
+milky_way.return_planets
+
+####Wave_2 & Wave_3 #####
+
+class SolarSystem
+
+  #use @planets instance variable
+  def initialize(planets)
+    @planets = Array.new(planets)
+  end
+
+  #method to add planet to list
+  def add(planet)
+    @planet = planet
+    @planets << planet
```

Comment:
You probably don't need to store the planet in `@planet` here. A quick search of your code shows that `@planet` is never used.

Instead you can just add it directly to `@planets`.